core/templates/hashfuncs.h

```diff
@@ -186,7 +341,9 @@ struct HashMapHasherDefault {
 	static _FORCE_INLINE_ uint32_t hash(const char32_t p_uchar) { return (uint32_t)p_uchar; }
 	static _FORCE_INLINE_ uint32_t hash(const RID &p_rid) { return hash_one_uint64(p_rid.get_id()); }
 
-	static _FORCE_INLINE_ uint32_t hash(const StringName &p_string_name) { return p_string_name.hash(); }
+	static _FORCE_INLINE_ uint32_t hash(const StringName &p_string_name) {
+		return p_string_name.hash();
+	}
```

Comment:
I guess this and the one above were wrongly changed by buggy clang-format 14.0.0?  https://github.com/llvm/llvm-project/issues/54901

Reply:
This was a leftover from testing some other hash functions which required several lines of code. Just cleaned it up.